### PR TITLE
MP#5 Setup product page lijuan

### PR DESCRIPTION
**Pull request by jenniferhou762**
Add my stuff.

### CODE REVIEW

File: companies/lijuan/*scratch* [hunk 1]
```diff
@@ -0,0 +1 @@
+Warning (emacs): PHP function descriptions not loaded. Try M-x php-extras-generate-eldoc
```
Comment (KunSu) [inline companies/lijuan/*scratch*]: Delete unrelated file

File: companies/lijuan/bubble-tea.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php include 'header.php';?>
```
Comment (KunSu) [inline companies/lijuan/bubble-tea.php]: Let's follow the same standard by copy and paste my code first. Otherwise, you need to implement all the features by yourself or at least implement all the HTML & CSS by yourself. Your product page should look exactly like mine. Or it will result 6 times workload for you.

Comment (KunSu) [inline companies/lijuan/bubble-tea.php]: all naming is allow spacing and please follow the naming convention as other teammates do. Otherwise again will increase your workload badly. including images naming.

**Review by KunSu — CHANGES_REQUESTED** [review]
Please resize your images in the index.php with the same width and height, and address the comment.

File: companies/lijuan/bubble-tea.php (hunk 2 above)
Comment (jenniferhou762) [reply companies/lijuan/bubble-tea.php]: You can take a look at the header.php file, I just put all your code there, so I can reuse it.

Comment (jenniferhou762) [reply companies/lijuan/bubble-tea.php]: You can take a look at the header.php file, I just put all your code there, so I can reuse it.

**Comment by KunSu** [comment]
https://github.com/KunSu/CMPE272MarketPlace/pull/21
Check this out. Look for the difference between your code and her code. Do not change my product page template unless you want to redo everything on your own.

**Comment by KunSu** [comment]
@jenniferhou762 You need to copy and paste one of my product's page under `companies/kun`. It means all you need to do is copy and paste my code and only change the content itself. So all product pages should look exactly the same except the image and some wording. 

If you insist on working on your own style that is fine for me, however, you will need to do all the work alone since other teammates' work might not fit into your pages.

**Comment by jenniferhou762** [comment]
@KunSu do you want to take another look?

**Review by KunSu — COMMENTED** [review]
Looks fine for me. There is only one place needs a quick fix. for each page under your companies except the first one is missing the `</section >` at the end of the file. After that, we can merge it.

**Review by KunSu — APPROVED** [review]
LGTM
